app/models/rental.py

```diff
@@ -1,4 +1,42 @@
 from app import db
+from app.models.customer import Customer
+from app.models.video import Video
+from datetime import date, timedelta
 
 class Rental(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
\ No newline at end of file
+    customer_id = db.Column(db.Integer, db.ForeignKey('customer.id'), primary_key=True, nullable=False)
+    video_id = db.Column(db.Integer, db.ForeignKey('video.id'), primary_key=True, nullable=False)
+    checkout_date = db.Column(db.DateTime)
+    checkout_status = db.Column(db.Boolean, default=True)
+    
+    def calculate_due_date(self):
+        """Calculates due date as seven days from today. """
+        due_date = str(self.checkout_date + timedelta(days=7))
+        return due_date
+
+    def get_count_checkedout_for_specific_video(self,video_id):
+        rentals = Rental.query.filter_by(video_id=video_id).filter_by(checkout_status=True).all()
+        return len(rentals)
```

Comment:
Methods like this might be best in the `Video` class, as you're getting information about a specific video.